datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -143,6 +150,17 @@ export default function UserInfoSideBar({ sideBarData, refetch }: Props) {
                         </Space>
                     </SocialDetails>
                     <Divider className="divider-aboutSection" />
+                    { countryCode && (
+                        <LocationSection>
+                            Location 
+                            <br />
+                                <LocationSectionText>
+                                    <img src={GlobeIcon} alt="Manage Users" style={{ display: 'inline' }} /> &nbsp;
+                                    {countries[countryCode].name}
```

Comment:
This is nice, but we should have some try-catch handling when resolving the country display name I think. Maybe you can do the lookup inside the countries in a utility method that returns Null / Undefined if it cannot find one.

Else we can have render-time uncaught exceptions here, which will cause problems.